config/app.default.php

```diff
@@ -244,6 +249,8 @@
              * which is the recommended value in production environments
              */
             //'init' => ['SET GLOBAL innodb_stats_on_metadata = 0'],
+            
+            'url' => env('DATABASE_URL', ''),
```

Comment:
`env()` also prases whatever APACHE (and nginx) supply, doesn't it?

If that is the case we should stick to a prefix to make sure that that stuff did not come from the webserver such as:
`CAKE_*` or `CONF_*`. This will help twofold:

- Make transparent where things came from.
- Make sure there are no accidental clashes between the environment the server sets, and the environment you set in your env file

This is just for the convention out of the box. You can still customize the app config to your desires and utilize other env pseudo-constants.